examples/03_promise/src/components/Counter.tsx

```diff
@@ -26,9 +26,12 @@ const Message = ({
 }: {
   count: number;
   delayedMessage: Promise<string>;
-}) => (
-  <ul>
-    <li>count: {count}</li>
-    <li>delayedMessage: {delayedMessage as any}</li>
-  </ul>
-);
+}) => {
+  const message = use(delayedMessage);
+  return (
+    <ul>
+      <li>count: {count}</li>
+      <li>delayedMessage: {message}</li>
```

Comment:
Hm, ARAIU, React should internally `use` a promise which is directly put in JSX.

Reply:
I'm not sure why...🤔 maybe ask React member to get some help?